<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>NGS</title>
  <meta name="description" content="">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <meta name="author" content="HirajDey">

  <link rel="apple-touch-icon" href="apple-touch-icon.png">
  <link rel="shortcut icon" href="https://2bzyy32beq443ip8qe29tijy-wpengine.netdna-ssl.com/wp-content/themes/medgenome/images/favicon.ico">
  <!-- Place favicon.ico in the root directory -->

  <link rel="stylesheet" href="css/normalize.min.css">
  <link rel="stylesheet" href="css/main.css">
  <link rel="stylesheet" href="css/icons.css">
  <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">

</head>
<body>
  <!--[if lte IE 9]>
    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
  <![endif]-->

<header class="site-header" id="section4">
  <div class="site-logo-wrap">
    <div class="site-logo">
      <img src="https://2bzyy32beq443ip8qe29tijy-wpengine.netdna-ssl.com/wp-content/uploads/2017/05/logo.png" alt="medgenome logo">
    </div>
  </div>
  
  <div class="hamburger-btn" data="hamburger-menu">
    <span></span>
    <span></span>
    <span></span>
  </div>

  <nav class="site-nav container">
    <ul>
      <li><a href="#">About</a></li>
      <li><a href="#">Immuno-oncology</a></li>
      <li><a class="active" href="index.html">Ngs Services</a></li>
      <li><a href="#">Data</a></li>
      <li><a href="#">Events</a></li>
      <li><a href="#">Press</a></li>
      <li><a href="#">Publications</a></li>
      <li><a href="#">Resources</a></li>
      <li><a href="medgenome-careers.html">Careers</a></li>
      <li><a href="#">Contact</a></li>
    </ul>
  </nav>

  <div class="contact-box">
    <div class="contact-us">
      <a href="https://www.medgenome.com/research/"><span></span>US</a>
    </div>
    <div class="contact-email-phone">
      <a href="mailto:research@medgenome.com"><i class="icon-envelope"></i>research@medgenome.com</a>
      <a href="tel:+918884400954"><i class="icon-phone"></i>(888) 440-0954</a>
    </div>
  </div>
</header><!-- eo site-header  -->

<section class="medgenome-bnr"></section><!-- eo medgenome-bnr -->

<section class="services-contant-section container">
  <div class="services-contant-box">
    <div class="unique-unlock-unprecedented-row">
      <div class="uuu-col">
        <div class="uuu-icon">
          <img src="img/unique-ios-icon.png" alt="Unique Immuno-oncology Solutions">
        </div>
        <div class="uuu-text">
          <h2>Unique Immuno-oncology Solutions</h2>
        </div>
      </div>
      <div class="uuu-col">
        <div class="uuu-icon">
          <img src="img/unlock-rif-icon.png" alt="Unlock Rich Insights Fast">
        </div>
        <div class="uuu-text">
          <h2>Unlock Rich Insights Fast</h2>
        </div>
      </div>
      <div class="uuu-col">
        <div class="uuu-icon">
          <img src="img/unprecedented-apgd-icon.png" alt="Unprecedented Access to Phenotype & Genotype Data">
        </div>
        <div class="uuu-text">
          <h2>Unprecedented Access to Phenotype & Genotype Data</h2>
        </div>
      </div>
    </div><!-- unique-unlock-unprecedented-row -->
    
    <div class="breadcrumbs-row">
      <ul>
        <li><a href="index.html">Home</a><span> | </span></li>
        <li><a href="index.html" class="active">NGS</a></li>
      </ul>
    </div><!-- eo breadcrumbs-row -->

    <div class="mdgo-tab-main-wrap">
      <div class="mdgo-tab-btn smooth-up1">
        <h3>NGS Services</h3>
        <span class="tab-arrow tab-arrow-top"></span>
      </div>
      <div class="mdgo-tab-content-wrap ngs-services-content">
        <p>At MedGenome we offer an end to-end solution for Next-generation sequencing as a service for a wide range of sample types ( DNA, RNA and specialized samples like T&B cells) as shown in Figure 1. We provide the user flexibility by providing them the option to use our services from  end-to-end or simply request services at any step of the workflow. To learn more about our services click on the tabs in the workflow below.</p>
  
        <div class="ngs-inner-tab-wrap">
          <ul class="ngs-inner-tab-btn">
            <li class="order-2"><a href="#" trgt_link="libraryPrep"><span>Library Preparation</span><span class="down-Arrow"></span></a></li>
            <li class="order-3"><a href="#" trgt_link="sequencing"><span>Sequencing</span><span class="down-Arrow"></span></a></li>
            <li class="order-4"><a href="#" trgt_link="bioinformaticsAnalysis"><span>Bioinformatics Analysis</span><span class="down-Arrow"></span></a></li>
            <li class="order-1"><a class="active" href="#" trgt_link="samplePreparation">Sample Preparation <br> & Extraction <span class="down-Arrow"></span></a></li> 
          </ul>

          <div class="ngs-inner-tab-content-wrap active" trgt_item="samplePreparation">
            <div class="table-container table-scroll-mob">
              <table>
                <thead>
                  <tr>
                    <th>Sample Type</th>
                    <th>DNA (WGS/WES/Targeted)</th>
                    <th>RNA (Transcriptome)</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>Extracted Nucleic Acid</td>
                    <td>Minimum: 200 ng (WGS/WES) 50 ng (Targeted)</td>
                    <td>Minimum: 100 ng (DV<sub>200</sub> > 30%)</td>
                  </tr>
                  <tr>
                    <td>Frozen cell pellets</td>
                    <td>Minimum 1 x 10<sup>6</sup> cells (ship on dry ice)</td>
                    <td>Minimum 1 x 10<sup>6</sup> cells (ship on dry ice)</td>
                  </tr>
                  <tr>
                    <td>Frozen tissues</td>
                    <td>30 mg frozen tissue or 20 mg stabilized tissue (ship on dry ice)</td>
                    <td>30 mg frozen tissue or 20 mg stabilized tissue (ship on dry ice)</td>
                  </tr>
                  <tr>
                    <td>FFPE slides</td>
                    <td>5 slides (10 μM thick) 250 mm<sup>2</sup> surface area</td>
                    <td>8 slides (10 μM thick) 250 mm<sup>2</sup> surface area</td>
                  </tr>
                  <tr>
                    <td>Curls</td>
                    <td>5 curls (10 μM thick)</td>
                    <td>8 curls (10 μM thick)</td>
                  </tr>
                  <tr>
                    <td>Single-cell</td>
                    <td>Fresh or cells in sample prep agent</td>
                    <td>Fresh or cells in sample-prep agent</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <p>Lower RNA & DNA amounts can be used for special projects </p>
          </div><!-- eo ngs-inner-tab-content-wrap -->

          <div class="ngs-inner-tab-content-wrap" trgt_item="libraryPrep">
            <div class="table-container table-scroll-mob">
              <table>
                <thead>
                  <tr>
                    <th>Service Type</th>
                    <th>Kit Used</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td><span>RNA (whole transcriptome)</span> <span>Transcribed exonic regions</span> <span>Small RNA</span></td>
                    <td><span>TrueSeq RNA V2, TrueSeq Stranded (No Capture probes used)</span><span>TrueSeq RNA Access</span> <span>Trilink (No Capture probes used)</span> <span>NEB (No Capture probes used)</span></td>
                  </tr>
                  <tr>
                    <td>Whole exome (WES)</td>
                    <td><span>SureSelect V5</span><span>SureSelect V5+UTR</span><span>TrueSeq Exome</span></td>
                  </tr>
                  <tr>
                    <td>Whole genome (WGS)</td>
                    <td><span>TrueSeq NANO</span> <span>KAPA (No Capture probes used)</span></td>
                  </tr>
                  <tr>
                    <td>MethylSeq</td>
                    <td>Agilent Sure Select XT Methyl-Seq Target Enrichment kit</td>
                  </tr>
                  <tr>
                    <td>TCR sequencing</td>
                    <td>ImmunoSeq, SMARTer TCR Profiling</td>
                  </tr>
                  <tr>
                    <td>Panel sequencing</td>
                    <td>Custom design (Nimblegen, IDT)</td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div><!-- eo ngs-inner-tab-content-wrap -->

          <div class="ngs-inner-tab-content-wrap" trgt_item="sequencing">
            <div class="table-container table-scroll-mob">
              <table>
                <thead>
                  <tr>
                    <th>Sample Type</th>
                    <th>Illumina Sequencing Platform</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>WGS/WES/Targeted</td>
                    <td>Illumina NovaSeq™ 6000, HiSeq 2500</td>
                  </tr>
                  <tr>
                    <td>Whole-transcriptome profiling</td>
                    <td>Illumina NovaSeq™ 6000, HiSeq 2500, MiSeq</td>
                  </tr>
                  <tr>
                    <td>TCR profiling</td>
                    <td>Illumina MiSeq, NovaSeq™ 6000 ( high throughput single-cell )</td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div><!-- eo ngs-inner-tab-content-wrap -->

          <div class="ngs-inner-tab-content-wrap bioinformatics-analysis-content" trgt_item="bioinformaticsAnalysis">
            <h3>WGS Germline variant calling Workflow</h3>
            
            <div class="wgs-workflow">
              <div class="lightbox-click">
                <img src="img/wgs-workflow.png" alt="WGS Germline variant calling Workflow">
              </div>

              <div class="lightbox-target">
                <div class="lightbox-inner-wrap">
                  <div class="lightbox-img">
                    <img src="img/wgs-workflow.png"/>
                  </div>
                </div>
                <span class="lightbox-close"></span>
              </div>
            </div>


            <h3>MedGenome VariMAT – Variant deep annotation program</h3>
            <ul class="varimat-list">
              <li>Multi-stage annotation pipeline for more accurate, rapid discovery and annotation of germline and somatic variants in samples.</li>
              <li>One of the most comprehensive variant annotation tools in the world.</li>
              <li>Integrated with disease databases - OMIM, HGMD, SwissVar, ClinVar, GWAS, OncoMD</li>
              <li>Integrated with ExAC, 1000G phase 3, dbSNP, EVS, 1KJP, MedVarDb databases.</li>
            </ul>

            <h3>Somatic Variant Calling and Annotation Pipeline</h3>

            <div class="somatic-pipeline">
              <div class="lightbox-click">
                <img src="img/somatic-variant-pipeline.png" alt="Somatic Variant Calling and Annotation Pipeline">
              </div>

              <div class="lightbox-target">
                <div class="lightbox-inner-wrap">
                  <div class="lightbox-img">
                    <img src="img/somatic-variant-pipeline.png" alt="Somatic Variant Calling and Annotation Pipeline">
                  </div>
                </div>
                <span class="lightbox-close"></span>
              </div>
            </div>

            <h3>Whole-Transcriptome Analysis Pipeline</h3>

            <div class="whole-transcriptome-pipeline">
              <div class="lightbox-click">
                <img src="img/whole-transcriptome-pipeline.png" alt="whole transcriptome analysis pipeline">
              </div>

              <div class="lightbox-target">
                <div class="lightbox-inner-wrap">
                  <div class="lightbox-img">
                    <img src="img/whole-transcriptome-pipeline.png" alt="whole transcriptome analysis pipeline">
                  </div>
                </div>
                <span class="lightbox-close"></span>
              </div>
            </div> 
      
          </div><!-- eo ngs-inner-tab-content-wrap -->
        </div><!-- eo ngs-inner-tab-wrap -->

        <p>Figure 1 : Workflow of the NGS Services provided at MedGenome for a wide range of sample types. Shown in this diagram, we highlight the steps taken to validate  successful sample preparation, library generation, sequencing and data analysis for successful completion of a project. Click on each tab to learn more about the options available, steps followed and the QC reports generated.</p>
      </div>
      
      <div class="mdgo-tab-btn smooth-up2">
        <h3>Whole Genome / Whole  Exome Sequencing</h3>
        <span class="tab-arrow"></span>
      </div>
      <div class="mdgo-tab-content-wrap whole-genome-content">
        <div class="whole-genome-trans-tcr-nav">
          <ul>
            <li class="active"><a href="#">About</a></li><li><a href="qc-reports.html">QC Report</a></li><li><a href="https://www.medgenome.com/ngs-bioinformatics-molecular-biology-services/">Informatics</a></li><li><a href="https://www.medgenome.com/publications/?act=whitepaper">White paper</a></li><li class="requestQuotePopupClick"><a href="#">Place and order</a></li>
          </ul>
        </div>
        <p>Whole Genome and Exome Sequencing using NGS have been widely accepted to speed up and reduce the cost of sequencing genomes for basic research as well as use of genomic data for a wide range of applications : genome wide association studies for complex diseases, variant calling to identify clinically actionable mutations and other specialized areas like identification of mutations that accumulate and give rise of tumor neo-antigens and shared antigens. At MedGenome we routinely perform Whole Genome Sequencing and  Exome Sequencing and Analyses using best practices.</p>
        <div class="medgenome-row">
          <div class="medgenome-col">
            <div class="table-container table-scroll-mob whole-exome-table1">
              <table>
                <thead>
                  <tr>
                    <th>Sample Type</th>
                    <th>DNA (WGS/WES/Targeted)</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>Extracted Nucleic Acid</td>
                    <td>Minimum: 200 ng (WGS/WES) 50 ng (Targeted)</td>
                  </tr>
                  <tr>
                    <td>Frozen Cell Pellets</td>
                    <td>Minimum 1 x 10<sup>6</sup> cells (ship on dry ice)</td>
                  </tr>
                  <tr>
                    <td>Frozen Tissues</td>
                    <td>30 mg frozen tissue or 20 mg stabilized tissue (ship on dry ice)</td>
                  </tr>
                  <tr>
                    <td>FFPE Slides</td>
                    <td>5 slides (10 μM thick) 250 mm2 surface area</td>
                  </tr>
                  <tr>
                    <td>Curls</td>
                    <td>5 curls (10 μM thick)</td>
                  </tr>
                  <tr>
                    <td>Single-cell</td>
                    <td>Fresh or cells in sample prep agent</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <p>Figure 1: a) Table of all the sample types used at MedGenome for WGS/WES</p>
          </div>
          <div class="medgenome-col">
            <div class="table-container table-scroll-mob standardAnlOff">
              <table>
                <thead>
                  <tr>
                    <th>Standard Analysis Offering</th>
                    <th>Advance Analysis Offering</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>Data QC (Read orientation, quality distribution, base distribution and GC distribution for all reads)</td>
                    <td>CNV analysis</td>
                  </tr>
                  <tr>
                    <td>Alignment to the reference genome</td>
                    <td>Pathway analysis</td>
                  </tr>
                  <tr>
                    <td>Coverage analysis</td>
                    <td>Deep variant annotation</td>
                  </tr>
                  <tr>
                    <td>Variant calling (germline/somatic calling)</td>
                    <td>Mutational burden analysis</td>
                  </tr>
                  <tr>
                    <td>Variant Annotation</td>
                    <td>Variant interpretation</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <p>Figure 1: b) Table shows deliverables for analyses of WGS/WES datasheets</p>
          </div>
        </div>
        
        <div class="ngs-inner-tab-wrap genome-exo-seq-inner-wrap">
          <h3>WGS germline calling validation</h3>
          <p>Total number of SNPs and Indels predicted using different flavour of GATK on two different datasets</p>
          
          <div class="table-container table-scroll-mob wgs-germline-table">
              <table>
                <thead>
                  <tr>
                    <th>Dataset</th>
                    <th>Germline Variant Caller</th>
                    <th>#of SNPs</th>
                    <th>#of Indels</th>
                    <th>Total</th>
                  </tr>
                </thead>
                <tbody>
                  <tr class="wgs-gtr-gray">
                    <td>Gold Standard</td>
                    <td>GiAB</td>
                    <td>3,193,799</td>
                    <td>391,621</td>
                    <td>3,585,420</td>
                  </tr>

                  <tr>
                    <td rowspan="5">Illumina HiSeq X10</td>
                    <td>GATK - Lite - 2.3.9 -UC</td>
                    <td>3,191,291</td>
                    <td>380,768</td>
                    <td>3,572,059</td>
                  </tr>
                  <tr class="wgs-gtr wgs-gtr-gray">
                    <td>Sentieon - 3.5 -HC</td>
                    <td>3,213,528</td>
                    <td>393,803</td>
                    <td>3,607,331</td>
                  </tr>
                  <tr class="wgs-gtr">
                    <td>Sentieon - 3.5 -UC</td>
                    <td>3,230,950</td>
                    <td>379,220</td>
                    <td>3,610,170</td>
                  </tr>
                  <tr class="wgs-gtr wgs-gtr-gray">
                    <td>GATK - 3.6 -HC</td>
                    <td>3,184,106</td>
                    <td>388,192</td>
                    <td>3,572,298</td>
                  </tr>
                  <tr class="wgs-gtr">
                    <td>GATK - 3.6 -UC</td>
                    <td>3,191,477</td>  
                    <td>376,688</td>
                    <td>3,568,165</td>
                  </tr>

                  <tr class="wgs-gtr-gray">
                    <td rowspan="5">Illumina HiSeq 2500 PublicData (NIST)</td>
                    <td>GATK - Lite - 2.3.9 -UC</td>
                    <td>3,198,322</td>
                    <td>383,952</td>
                    <td>3,582,274</td>
                  </tr>
                  <tr class="wgs-gtr">
                    <td>Sentieon- 3.5 -HC</td>
                    <td>3,215,682</td>
                    <td>397,577</td>
                    <td>3,613,259</td>
                  </tr>
                  <tr class="wgs-gtr wgs-gtr-gray">
                    <td>Sentieon - 3.5 -UC</td>
                    <td>3,242,040</td>
                    <td>386,397</td>
                    <td>3,628,437</td>
                  </tr>
                  <tr class="wgs-gtr">
                    <td>GATK - 3.6 -HC</td>
                    <td>3,186,676</td>
                    <td>393,300</td>
                    <td>3,579,976</td>
                  </tr>
                  <tr class="wgs-gtr wgs-gtr-gray">
                    <td>GATK - 3.6 -UC</td>
                    <td>3,198,478</td>
                    <td>383,917</td>
                    <td>3,582,395</td>
                  </tr>

                </tbody>
              </table>
          </div>
          <div class="medgenome-row">
            <div class="medgenome-col">
              <div class="metrics-btn">SNP Metrics</div>
              <div class="metrics-chart-box">
                <div class="lightbox-click">
                  <img src="img/snp-metrics.png" alt="snpMetrics">
                </div>
                <div class="lightbox-target">
                  <div class="lightbox-inner-wrap">
                    <div class="lightbox-img">
                      <img src="img/snp-metrics.png" alt="snpMetrics">
                    </div>
                  </div>
                  <span class="lightbox-close"></span>
                </div>
              </div>
            </div>
            <div class="medgenome-col">
              <div class="metrics-btn">InDel Metrics</div>
              <div class="metrics-chart-box">
                <div class="lightbox-click">
                  <img src="img/indel-metrics.png" alt="indelMetrics">
                </div>
                <div class="lightbox-target">
                  <div class="lightbox-inner-wrap">
                    <div class="lightbox-img">
                      <img src="img/indel-metrics.png" alt="indelMetrics">
                    </div>
                  </div>
                  <span class="lightbox-close"></span>
                </div>
              </div>
            </div>
          </div>

          <p class="chart-text">Figure 2 : Validation of WGS/WES pipelines using Gold Standard datasets</p>

          <ul class="snv-nist-text">
            <li><p>Sensitivity for SNV detection of our pipeline exceeded 99% at 30X-50X coverage and was over 99% for the detection of Indels at 50X coverage.</p></li>
            <li><p>The NIST dataset provided the best sensitivity and precision primarily due to higher sequencing depth (~50X) and PCR-free library.</p></li>
            <li><p>Our study on NA12878 gold standard results shows that our results are at par with industry standard and matches FDA challenge results.</p></li>
          </ul>
        </div><!-- ngs-inner-tab-wrap -->

        <p class="">We have recently added the 10X Chromium platform to our offerings and can perform single-cell Whole Genome Sequencing, and analysis as well as <i>de-novo</i> analysis using the 10X software solutions.</p>

      </div>

      <div class="mdgo-tab-btn smooth-up3">   
        <h3>Whole Transcriptome Sequencing</h3>
        <span class="tab-arrow"></span>
      </div>
      <div class="mdgo-tab-content-wrap whole-transcriptome-content">
        <div class="whole-genome-trans-tcr-nav">
           <ul>
            <li class="active"><a href="#">About</a></li><li><a href="qc-reports.html">QC Report</a></li><li><a href="https://www.medgenome.com/ngs-bioinformatics-molecular-biology-services/">Informatics</a></li><li><a href="https://www.medgenome.com/publications/?act=whitepaper">White paper</a></li><li class="requestQuotePopupClick"><a href="#">Place and order</a></li>
          </ul>
        </div>
        <p>Next generation sequencing enables gene expression analysis from normal and disease tissues with high-sensitivity. Profiling of coding and non-coding RNAs is important in revealing molecular mechanisms of development and disease. With the several options that exist for gene expression profiling, it is also possible to perform whole transcriptome profiling ranging from a single-cell to thousands of cells in a sample. In addition, current methodologies allow for profiling of transcriptomes from intact RNA and
        RNA degraded due to the storage and isolation conditions (such as FFPE processed tissue blocks, cell-free nucleic acids etc).  At MedGenome we offer solutions For performing gene expression analysis from a wide-range of sample types. Shown in table 1 is a range of sample types that we process for Whole transcriptome profiling.</p>

        <div class="ngs-inner-tab-wrap genome-exo-seq-inner-wrap">
          <h3>Sample requirements and library prep information</h3>
          <div class="table-container service-sample-library">
              <table>
                <thead>
                  <tr>
                    <th>Service</th>
                    <th>Sample Requirements</th>
                    <th>Library Kit Used</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td>RNA Sequencing</td>
                    <td><span>Recommended: 1 μg (RIN>8)</span> <span>Minimum: 50 ng (RIN < 5)</span> <span>Single-cell solutions available</span></td>
                    <td><span>TrueSeq RNA V2, TrueSeq Stranded (No Capture probes used)</span> <span>TrueSeq RNA Access</span> <span>Trilink (No Capture probes used)</span> <span>NEB (No Capture probes used)</span></td>
                  </tr>
                </tbody>
              </table>
          </div>

          <h3 class="mdg-mt">RNA analysis deliverables</h3>
          <div class="medgenome-row">
            <div class="medgenome-col">
              <div class="table-container rna-analysis-delvr">
                <table>
                  <thead>
                    <tr>
                      <th>Standard Analysis Offerings</th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>Data QC (Read orientation, quality distribution, base distribution and GC distribution for all reads)</td>
                    </tr>
                    <tr>
                      <td>Alignment to the reference genome & transcriptome</td>
                    </tr>
                    <tr>
                      <td>Gene and Isoform expression estimation in RPKM / Read count per gene</td>
                    </tr>
                    <tr>
                      <td>Statistical Analysis including p-values and coefficient of variance (CV) between replicates</td>
                    </tr>
                    <tr>
                      <td>Gene/Transcript expression levels in Excel</td>
                    </tr>
                    <tr>
                      <td>Differential Expression Comparison</td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
            <div class="medgenome-col">
              <div class="table-container rna-analysis-delvr">
                <table>
                  <thead>
                    <tr>
                      <th>Advanced Analysis Offerings</th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>Gene fusion detection</td>
                    </tr>
                    <tr>
                      <td>Wiggle file for gene expression visualization on genome browser</td>
                    </tr>
                    <tr>
                      <td>Pathway analysis</td>
                    </tr>
                    <tr>
                      <td>SNV & InDel</td>
                    </tr>
                    <tr>
                      <td>Splicing analysis</td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
        </div><!-- ngs-inner-tab-wrap -->

        <p>In addition to the NGS library preparation solutions, we offer extensive  informatics solutions for whole-transcriptome profiling
      that ranges from standard gene expression analyses to advanced analyses of splicing, pathways, visualization and tertiary
      analyses to reveal heterogeneity and cell-types in a population as well tumor-microenvironment analyses. In addition to that we 
      have solutions using machine-learning approaches that utilize gene expression data to predict biomarkers and classify tissue
      types and diseases.</p>
      </div>

      <div class="mdgo-tab-btn smooth-up4">
        <h3>TCR Sequencing</h3>
        <span class="tab-arrow"></span>
      </div>
      <div class="mdgo-tab-content-wrap tcr-sequencing-content">
         <div class="whole-genome-trans-tcr-nav">
          <ul>
            <li class="active"><a href="#">About</a></li><li><a href="qc-reports.html">QC Report</a></li><li><a href="https://www.medgenome.com/ngs-bioinformatics-molecular-biology-services/">Informatics</a></li><li><a href="https://www.medgenome.com/publications/?act=whitepaper">White paper</a></li><li class="requestQuotePopupClick"><a href="#">Place and order</a></li>
          </ul>
        </div>
        <p>T-cells are the core components of our adaptive immune system. Once activated, they can directly kill cells that are foreign (cytolytic T-cells) or perform helper function (helper T-cells) to activate B-cells to make antibodies against foreign antigen. The activation of T-cells involve recognition of MHC-peptide complex by the T-cell receptors (TCR). Humans carry >10<sup>9</sup> T-cells, each expressing a unique TCR. This highly diverse repertoire of T-cells has the ability to recognize peptides originating from foreign elements such as invading pathogens and cancer cells. Each TCR recognize peptides in complex with MHC proteins presented on the surface of antigen presenting cells. Productive T-cell activation results in the clonal expansion of a specific T-cell and this expansion can be accurately determined by TCR sequencing.</p>

        <p>TCR profiling holds great potential not only for understanding the mechanisms of development of the normal immune response, but also in providing insights into disease mechanisms and development of new therapeutics and treatment modalities in infectious diseases, autoimmunity and in immuno-oncology<sup>1</sup>. However identification of all potential clonotypes in a diverse repertoire of TCRs requires sensitive methods of detection<sup>2,3</sup>. Next-generation sequencing (NGS) technologies have recently enabled accurate detection of TCRs, and in combination with other assays allow for the assessment of the TCR repertoire in patients- which in turn is a proxy for patient prognosis and response. At MedGenome , we provide three different NGS based methods of profiling the TCR repertoire : immunoSEQ™ assay (Adaptive Biotechnologies), SMARTer® TCR Profiling Kit (Takara Bio USA Inc), and Single-cell V(D)J Immune Profiling solution (10X™ Genomics Inc) respectively.  To learn more about the different applications and sample data generated using TCR Seq at MedGenome read the <a class="highlightonhov" href="https://www.medgenome.com/publications/?act=whitepaper">white paper.</a></p>

        <div class="major-application-tcr-row">
          <div class="major-app-heading">
            <h3>Major Applications of TCR Repertoire Profiling</h3>
          </div>
          <!-- <div class="medgenome-row">
            <div class="medgenome-col">
              <div class="td tr-one"><span>Therapeutics</span></div>
              <div class="td tr-two"><span>Identify antigen-specific TCRs</span></div>
              <div class="td tr-three"><span>Generate engineered T cell for adoptive cell therapy</span></div>
            </div>
            <div class="medgenome-col">
              <div class="td tr-one"><span>Diagnostics</span></div>
              <div class="td tr-two"><span>Assay TCR clonality for autoimmune disorders</span></div>
              <div class="td tr-three"><span>Diagnostics for autoimmune & infectious diseases</span></div>
            </div>
            <div class="medgenome-col">
              <div class="td tr-one"><span>Treatment monitoring</span></div>
              <div class="td tr-two"><span>Assay TCR repertoire (tumor & autoimmune disorders)</span></div>
              <div class="td tr-three"><span>Identify biomarkers of response</span></div>
            </div>
          </div> -->

          <div class="major-application-col">
            <img src="img/major-application-tcr.jpg">
          </div>
        </div>
        
        <p>Figure 1. Overview of major applications of TCR Repertoire profiling in a) Therapeutics application: TCR Sequencing of lymphocytes to identify antigen specific TCRs to generate engineered T cells for adoptive cell therapy b) Diagnostics assay for clonality for auto immune disorders c) Treatment monitoring: to identify biomarkers of response.</p>
        
        <h3>Overview of the service offerings from Medgenome</h3>

        <div class="table-container table-scroll-mob">
          <table>
            <thead>
              <tr>
                <th>Name of offering</th>
                <th>Input type</th>
                <th>Amount of material neeeded</th>
                <th>Analysis method</th>
                <th>Information Obtained</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>immunoSEQ Kit (Adaptive Biotechnologies)</td>
                <td>gDNA</td>
                <td>300 ng gDNA/well (Human, Mouse)</td>
                <td>immunoSEQ Analyzer</td>
                <td>CDR3, clonotypes</td>
              </tr>
              <tr>
                <td>Single cell Immune profiling (10X Genomics)</td>
                <td>Isolated cells</td>
                <td>Single-cells (Human, Mouse)</td>
                <td>Loupe browser</td>
                <td>CDR3, α/β pairing and clonotypes V(D)J sequences</td>
              </tr>
              <tr>
                <td>SMARTer TCR profiling kit (Takara Bio USA)</td>
                <td>Isolated cells or RNA</td>
                <td>10 ng-3 ng/50-10,000 cells(Human, Mouse)</td>
                <td>MiXCR</td>
                <td>CDR3, V(D)J sequences α/β pairing(from SC Kit)</td>
              </tr>
             </tbody>
          </table>
        </div>

        <p>In addition to the TCR-Seq services, MedGenome offers advanced informatics solutions for TCR-Seq based prediction of prioritization of neo-epitopes for Cancer vaccine development using  OncoPeptVAC™</p>
      </div>

    </div><!-- mdgo-tab-main-wrap -->

  </div><!-- eo services-contant-box -->
</section><!-- eo services-contant-section -->


<section class="medgenome-form-section">  
  <div class="container">
    <div class="genomic-research-row">
      <h2>Considering genomic research?</h2>
      <h3>We're at your service</h3>
      <p>Contact MedGenome today to accelerate your NGS research quickly and cost-effectively</p>

      <a class="requestQuotePopupClick" href="#">Request a quote</a>
    </div>

    <form class="form-box validate-form" autocomplete="off" novalidate>
      <div class="form-row">
        <div class="form-col">
          <div class="form-control">
            <input class="inputEmpt fm-name"  type="text" name="Name" placeholder="Name">
            <span class="fm-mandatory">*</span>            
            <span class="fm-icon"><i class="icon-user"></i></span>
            <div class="error-alert">
              <span>please enter your name</span>
            </div>
          </div>
          <div class="form-control">
            <input class="inputEmpt fm-email" type="text" name="Email" placeholder="Email">
            <span class="fm-mandatory">*</span>
            <span class="fm-icon"><i class="icon-envelope"></i></span>
            <div class="error-alert">
              <span>please enter your valid email</span>
            </div>
          </div>
        </div>
        <div class="form-col">
          <div class="form-control">
            <input class="inputEmpt fm-organization" type="text" name="Organization" placeholder="Organization">
            <span class="fm-icon"><i class="icon-office"></i></span>
          </div>
          <div class="form-control">
            <input class="inputEmpt fm-phonenumber" type="text" name="Phone Number" placeholder="Phone Number">
            <span class="fm-icon"><i class="icon-phone"></i></span>
            <div class="error-alert">
              <span>please enter your phone number</span>
            </div>
          </div>
          <div class="select-control-wrap">
                  <select class="rw-select-box required nric-select-cnt">
                    <option value="Interest">Interest</option>
                    <option value="General inquiry">General inquiry</option>
                    <option value="Next Generation Sequencing services">Next Generation Sequencing services</option>
                    <option value="Bioinformatics services">Bioinformatics services</option>
                    <option value="Immuno-oncology research">Immuno-oncology research</option>
                    <option value="Diabetes research">Diabetes research</option>
                    <option value="Press inquiry">Press inquiry</option>
                    <option value="Career inquiry">Career inquiry</option>
                    <option value="Other">Other</option>
                  </select>
                <span class="fm-mandatory">*</span>    
                <div class="error-alert">
                  <span>please select your interest</span>
                </div>                
          </div>
        </div>
        <div class="form-col">
          <div class="form-control inquiry-control">
            <textarea name="Inquiry / Message"  rows="6" placeholder="Inquiry / Message"></textarea>
            <span class="fm-icon"><i class="icon-edit"></i></span>
          </div>
        </div>
      </div>
      <div class="submit-row">
        <div class="quiry-information">
          <div class="submit-error-text">
            <p>One or more fields have an error. Please check and try again.</p>
          </div>
          <div class="success-message">
            <p>Thank you for your interest in MedGenome.</p>
            <p>We are reviewing your request and will get back to you within 1 business day.</p>
          </div>
        </div>
        <input class="submitBtn" type="submit" name="submit" value="Submit">
      </div>
    </form>
  </div>
</section><!-- eo medgenome-form-section -->


<footer class="site-footer-section">
  <div class="footer-one">
    <div class="footer-row container">
      <div class="footer-col">
        <div class="site-logo">
          <img src="img/site-logo-footer.png" alt="medgenome logo">
        </div>
        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolorem, eum omnis. Autem fugit numquam amet ipsam nobis cupiditate, eaque maxime dignissimos deleniti quam, molestiae totam et laboriosam temporibus quis soluta?</p>
      </div>
      <div class="footer-col">
        <h3>Follow Us</h3>
        <a href="#"><i class="icon-facebook"></i> Facebook</a>
        <a href="#"><i class="icon-twitter2"></i> Twitter</a>
        <a href="#"><i class="icon-linkedin2"></i> Linkedin</a>
      </div>
      <div class="footer-col">
        <h3>Contact Info</h3>
        <a href="#"><i class="icon-envelope"></i> Contact@medgenome.com</a>
        <a href="#"><i class="icon-phone"></i> (888) 440-0954</a>
        <ul class="address">
          <li><i class="icon-map-marker"></i>Foster City</li>
          <li>Medgenome Inc</li>
          <li>438 Hatch Drive</li>
          <li>Foster City, CA 94404, USA</li>
        </ul>
      </div>
    </div>
  </div><!-- footer-one -->

  <div class="footer-two">
    <div class="footer-row container">
      <div class="footer-col">
        <ul class="footer-nav">
          <li><a href="#">About</a> <span>|</span> </li>
          <li><a href="#">Immuno-Oncology</a> <span>|</span> </li>
          <li><a href="#">Ngs</a> <span>|</span> </li>
          <li><a href="#">Events</a> <span>|</span> </li>
          <li><a href="#">Press</a> <span>|</span> </li>
          <li><a href="#">Publications</a> <span>|</span> </li>
          <li><a href="#">Contact</a></li>
        </ul>
      </div>
      <div class="footer-col">
        <p>2017 © MedGenome • All Rights Reserved</p>
      </div>
    </div>
  </div><!-- footer-two -->
</footer><!-- eo site-footer-section -->


<!-- !!!NOTE:- Don't delete this below div element class call PopUpWrap -->
<div class="PopUpWrap"></div>


  <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
  <script>window.jQuery || document.write('<script src="js/vendor/jquery-3.1.1.min.js"><\/script>')</script>

  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5/jquery.min.js"></script>
  <script src="js/plugins.js"></script>
  <script src="js/main.js"></script>


  <!-- Google Analytics: change UA-XXXXX-Y to be your site's ID. -->
  <script>
    window.ga=function(){ga.q.push(arguments)};ga.q=[];ga.l=+new Date;
    ga('create','UA-XXXXX-Y','auto');ga('send','pageview')
  </script>
  <script src="https://www.google-analytics.com/analytics.js" async defer></script>

</body>
</html>

